dts/bindings/mtd/jedec,spi-nor.yaml

```diff
@@ -24,13 +24,68 @@ properties:
   requires-ulbpr:
     type: boolean
     required: false
-    description: Indicates the device requires the ULBPR (0x98) command
+    description: |
+      Indicates the device requires the ULBPR (0x98) command.
 
       Some flash chips such as the Microchip SST26VF series have a block
-      protection register that initializes to write protected.  Use this
+      protection register that initializes to write-protected.  Use this
       property to indicate that the BPR must be unlocked before write
       operations can proceed.
 
+  has-dpd:
+    type: boolean
+    required: false
+    description: |
+      Indicates the device supports the DPD (0xB9) command.
+
+      Use this property to indicate the flash chip supports the Deep
+      Power-Down mode that is entered by command 0xB9 to reduce power
+      consumption below normal standby levels.  Use of this property
+      implies that the RDPD (0xAB) Release from Deep Power Down command
+      is also supported.  (On some chips this command functions as Read
+      Electronic Signature; see t-enter-dpd).
+
+  dpd-wakeup-sequence:
+    type: array
+    required: false
+    description: |
+      Specifies wakeup durations for devices without RDPD.
+
+      Some devices (Macronix MX25R in particular) wake from deep power
+      down by a timed sequence of CSn toggles rather than the RDPD
+      command.  This property specifies three durations measured in
```

Comment:
Can you specify here which time is which in DT property sequence? `<x y z>`

Reply:
I thought it was clear, but maybe the update is better.